@xen-orchestra/lite/src/views/new-vm/NewVmView.vue

```diff
@@ -0,0 +1,989 @@
+                  <optgroup :label="srName">
+                    <option v-for="vdi in VDIs" :key="vdi.$ref" :value="vdi.$ref">
+                      {{ vdi.name_label }}
+                    </option>
+                  </optgroup>
+                </template>
+              </FormSelect>
+            </div>
+          </div>
+          <!-- SYSTEM SECTION -->
+          <UiTitle>{{ $t('new-vm.system') }}</UiTitle>
+          <!-- <UiToggle v-model="vmState.toggle">{{ $t('new-vm.multi-creation') }}</UiToggle> -->
+          <div class="system-container">
+            <div class="col-left">
+              <UiInput v-model="vmState.vm_name" accent="brand" href="''">{{ $t('new-vm.vm-name') }}</UiInput>
+              <!-- <UiInput v-model="vmState.tags" :label-icon="faTags" accent="brand" href="''"> -->
+              <!-- {{ $t('new-vm.tags') }} -->
+              <!-- </UiInput> -->
+              <div>
+                <UiLabel accent="neutral" href="''">{{ $t('new-vm.boot-firmware') }}</UiLabel>
+                <FormSelect v-model="vmState.boot_firmware">
+                  <option v-for="boot in getBootFirmwares" :key="boot" :value="boot">
+                    {{ boot === undefined ? t('bios-default') : boot }}
+                  </option>
+                </FormSelect>
+              </div>
+              <div
+                v-if="vmState.boot_firmware === 'uefi' || templateHasBiosStrings"
+                v-tooltip="{
+                  placement: 'top-start',
+                  content:
+                    vmState.boot_firmware !== 'uefi'
+                      ? $t('new-vm.boot-firmware-bios')
+                      : $t('new-vm.boot-firmware-uefi'),
+                }"
+              >
+                <UiCheckbox v-model="getCopyHostBiosStrings" accent="brand" disabled>
+                  {{ $t('new-vm.copy-host') }}
+                </UiCheckbox>
+              </div>
+              <div v-else>
+                <UiCheckbox v-model="getCopyHostBiosStrings" accent="brand">
+                  {{ $t('new-vm.copy-host') }}
+                </UiCheckbox>
+              </div>
+            </div>
+            <div class="col-right">
+              <UiTextarea v-model="vmState.vm_description" accent="brand" href="''">
+                {{ $t('new-vm.vm-description') }}
+              </UiTextarea>
+              <!-- <UiInput v-model="vmState.affinity_host" accent="brand" href="''">{{ $t('new-vm.affinity-host') }}</UiInput> -->
+            </div>
+          </div>
+          <!-- MEMORY SECTION -->
+          <UiTitle>{{ $t('new-vm.memory') }}</UiTitle>
+          <div class="memory-container">
+            <UiInput v-model="vmState.vCPU" accent="brand" href="''">{{ $t('new-vm.vcpu') }}</UiInput>
+            <!-- TODO remove (GB) when we can use new selector -->
+            <UiInput v-model="ramFormatted" accent="brand" href="''">{{ $t('new-vm.ram') }} (GB)</UiInput>
```

Comment:
To temporarily fix CI error while waiting for the component:
```suggestion
            <UiInput v-model="ramFormatted" accent="brand" href="''">{{ `${$t('new-vm.ram')} (GB)` }}</UiInput>
```